assignment_3/Marksheet/src/Marksheet.java

```diff
@@ -137,23 +145,23 @@ public static void main(String[] args)
 			}
 		}
 		
-		//calls the Calculate_Average method 
-		float Average= marksheet.Calculate_Average(grades);
-		System.out.print("Average of the students' grades is = "+Average);
+		//calls the Calculate_average method 
+		float average= marksheet.calculateaverage(grades);
+		System.out.print("average of the students' grades is = "+average);
```

Comment:
First letter n a text message should be in upper case
System.out.print("**A**verage of the students' grades is = "+average);